INSTALLATION.md

```diff
@@ -152,12 +161,11 @@ Using Mongosh:          1.6.2
 
 For mongosh info see: https://docs.mongodb.com/mongodb-shell/
 
-...
-...
-...
-...
-
 ```
+8. In a separate terminal, run the "mongod" command to start the local instance of the database.
+9. Open MongoDB Compass and click on "Connect." You will now be able to access the graphical user interface of the local database.
+
+**NOTE**:These instructions are for Windows only you can do the same in macOS and linux with minor tweaks. This has been provided to give a brief overview for beginners to setup their own local instance.
```

Comment:
Where is the information regarding populating the Admin user?

1. The information here is very useful. https://github.com/PalisadoesFoundation/talawa-admin/pull/454/files
2. Also **ALL** the files referred to in the installation guide **must** be a part of the repository.